Update Fraud Prevention docs (#565)
swan-io/docs: docs/partnership/protections/fraud/types.mdx

:::note Tips to combat phishing, vishing, and smishing fraud
- Don't provide information or funds **urgently**.
- **Contact the organization directly**. Use a different communication channel to ask them about the suspicious message. For example, if they contacted you by phone, send an email. If you received a text message, call them.
- **Check links before you open them**. Fraudulent websites often use small URL changes to impersonate legitimate sites. [Double-check](../../../partnership/protections/fraud/#double-check) for typos, extra characters, letters, or numbers in the link. 
- **Open websites directly** in your browser instead of clicking links included in an email or text message.

> - Set up **multi-factor authentication (MFA)** or **two-factor authentication (2FA)** as an extra layer of security in case credentials are compromised.

Reviewer: ```suggestion
- Enable **multi-factor authentication (MFA)** or **two-factor authentication (2FA)** for added security if your credentials are compromised.
```
Simplify
